skyportal/handlers/api/news_feed.py

```diff
@@ -104,6 +114,39 @@ def fetch_newest(model):
                 for c in comments
             ]
         )
+        news_feed_items.extend(
+            [
+                {
+                    "type": "classification",
+                    "time": c.created_at,
+                    "message": f"New classification for {c.obj_id} added by {c.author.username}: {c.classification}",
+                    "source_id": c.obj_id,
+                }
+                for c in classifications
+            ]
+        )
+        news_feed_items.extend(
+            [
+                {
+                    "type": "spectrum",
+                    "time": s.created_at,
+                    "message": f"New spectrum taken with {s.instrument.name} uploaded for {s.obj_id}",
```

Comment:
Describe who uploaded the spectrum, 
```suggestion
                    "message": f"{s.owner.first_name} {s.owner.last_name} uploaded a {s.instrument.name} spectrum for {s.obj_id}",
```